### PR TITLE
[WIP] Style consolidation and deduplication

### PR DESCRIPTION
**Pull request by codecaaron**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (sonnynomnom) [inline .gitignore:1]: 😅

File: public/styles/shared.css [hunk 2]
```diff
@@ -1,325 +1,411 @@
...
+  padding: 4rem 2rem 0;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.section--content,
+.section--callout {
+  position: relative;
+  margin: auto;
+  padding: 5rem 0;
```
Comment (sonnynomnom) [inline public/styles/shared.css]: On the FAQ page, http://localhost:8080/faq.html:

The text "Let us know if you have other questions here." has some extra padding on top and bottom.

File: public/faq.html [hunk 3]
```diff
@@ -56,320 +71,304 @@
...
-          <a href="https://www.github.com/karaokenite" target="_blank">
-          <img class="social" src="https://cdn.glitch.com/b316bbdc-0b0c-4c6d-94fb-fffb37f510a9%2Fgithub-logo.svg?v=1597727187892"></a></div>
-       
-        <div class="column">
-          <a href="https://discord.gg/xUc8m25" target="_blank">
-          <img class="social" src="https://cdn.glitch.com/b316bbdc-0b0c-4c6d-94fb-fffb37f510a9%2Fdiscord%20logo.svg?v=1599600184242"></a>
+    <footer class="footer">
+      <div class="footer-container">
+        <div class="footer-left">
+          <span id="footer-name">KARAOKE NITE</span>
```
Comment (sonnynomnom) [inline public/faq.html:326]: Add `<br />` after `</span>`?

Similar to what we have in index.html.

File: public/about.html [hunk 4]
```diff
@@ -57,238 +70,234 @@
...
-      <br>
-      <br>
-      <br>
-      <br>
+    </aside>
+    <!-- New Footer (Sonny) -->
+    <footer class="footer">
+      <div class="footer-container">
+        <div class="footer-left">
+          <span id="footer-name">KARAOKE NITE</span>
```
Comment (sonnynomnom) [inline public/about.html:255]: Add `<br />` after `</span>`?

Similar to what we have in index.html.

File: public/blog.html [hunk 5]
```diff
@@ -56,349 +70,382 @@
...
-          <a href="https://www.github.com/karaokenite" target="_blank">
-          <img class="social" src="https://cdn.glitch.com/b316bbdc-0b0c-4c6d-94fb-fffb37f510a9%2Fgithub-logo.svg?v=1597727187892"></a></div>
-       
-        <div class="column">
-          <a href="https://discord.gg/xUc8m25" target="_blank">
-          <img class="social" src="https://cdn.glitch.com/b316bbdc-0b0c-4c6d-94fb-fffb37f510a9%2Fdiscord%20logo.svg?v=1599600184242"></a>
+    <footer class="footer">
+      <div class="footer-container">
+        <div class="footer-left">
+          <span id="footer-name">KARAOKE NITE</span>
```
Comment (sonnynomnom) [inline public/blog.html:403]: Add `<br />` after `</span>`?

Similar to what we have in index.html.

File: public/styles/scene.css [hunk 6]
```diff
@@ -128,13 +120,9 @@ kbd {
   position: absolute;
   font-size: 30px;
   text-transform: uppercase;
-  color: #ffffff;
+  color: var(--color-linen);
   font-weight: 700;
-  font-family: "Linotte";
-}
-
-#smiley {
```
Comment (sonnynomnom) [inline public/styles/scene.css:136]: I've been trying to figure this out for the last 20 minutes but can't figure out the root issue of this, but the smiley face in scene.html is gone.

karaokenite.co:

karaokenite co

here:

this_branch

Comment (codecaaron) [reply public/styles/scene.css:136]: Got it, position inheritance from `a`.  Going to remove the styles we are applying directly to tags too so this shouldn't happen.

**Pull request merged**